.github/workflows/build.yml

```
@@ -0,0 +1,33 @@
+name: Build
+on:
+  pull_request:
+    branches:
+      - main
```

@Vvaradinov needs to be changed to `master`
```suggestion
      - master
```